mithril-aggregator/src/services/stake_distribution.rs

```diff
@@ -169,10 +169,7 @@ impl StakeDistributionService for MithrilStakeDistributionService {
             .get_stakes(epoch)
             .await
             .map_err(|e| StakePoolDistributionServiceError::technical_subsystem(e.into()))?
-            .ok_or_else(|| StakePoolDistributionServiceError::Technical {
-                message: "The stake distribution should be at least an empty list.".to_string(),
-                error: None,
-            })?;
+            .unwrap_or_default();
```

Comment:
Is this the behavior we expect if we return default instead of an error here? :thinking: